app/models/post.rb

```diff
@@ -3,6 +3,12 @@ class Post < ApplicationRecord
   has_many :comments, foreign_key: 'posts_id'
   has_many :likes, foreign_key: 'posts_id'
 
+  validates :title, presence: true, length: { maximum: 250 }
+
+  validates :commentsCounter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
+  validates :likesCounter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Two of your post tests were failing because you are using the wrong attribute here. It should be `commentscounter` and `likescounter` based on what you have in your posts table. Kindly fix that.

  ![comm-1](https://user-images.githubusercontent.com/36542986/192111028-27a909c7-e514-4fc6-b393-15da88e380e1.jpg)